### PR TITLE
[maimaidx] Update to BUDDiES PLUS

### PR DESCRIPTION
**Pull request by Xinnoh**
- Includes content up to metropolis 8
- Updates chart constants
- In this version, most X.6 charts are now X+ difficulty, this has been updated
- The Metropolis 8 experts don't have their exact constants confirmed yet, this will need updating in the future.

Some charts have had their rating changed, but the constant hasn't been confirmed. (eg 夜に駆ける goes from 12+ to 13, but the constant is unconfirmed and is still 12.8 for now)

I looked online for the internal ID system and couldn't initially find anything so I just incremented based of the existing IDs
Then I looked at the merge options and found some stuff related to pulling IDs directly from the game, so yeah that will be something to keep in mind for when someone else updates it in the future. Changing them hopefully shouldn't be too much of an issue.
First time doing this so needs double checking

### CODE REVIEW

File: database-seeds/scripts/rerunners/maimaidx/merge-options.ts [hunk 1]
```diff
@@ -69,8 +70,7 @@ interface MusicXML {
 }
 
 function calculateLevel(data: Pick<NotesData, "level" | "levelDecimal">) {
-	// TODO: Change levelDecimal condition to 6 when updating to BUDDiES+
-	return `${data.level}${data.levelDecimal >= 7 && data.level >= 7 ? "+" : ""}`;
+	return `${data.level}${data.levelDecimal >= 6 && data.level >= 6 ? "+" : ""}`;
```
Comment (beerpiss) [inline database-seeds/scripts/rerunners/maimaidx/merge-options.ts:73]:
```suggestion
	return `${data.level}${data.levelDecimal >= 6 && data.level >= 7 ? "+" : ""}`;
```

While the condition for `+` levels have been changed from `.7` to `.6`, the lowest level with a `+` is still 7. There is no level 6+.

**Comment by zkldi** [comment]
cc @j1nxie, @beerpiss; is this ok to merge?

**Comment by Xinnoh** [comment]
Yeah also misunderstood tachiID, should be fine there.

we discussed a bit in discord about data accuracy. I'll double check some of these constants and update the metropolis experts

**Comment by zkldi** [comment]
```
 [VALIDATING DUPES] charts-maimaidx
[ERR] charts-maimaidx.json | EasyPop - ハッピーシンセサイザ (16) - Single DX Advanced (52e08a52c584ece3245be2aa4a5ebb114077d5e3) | Is duplicate on chartID:52e08a52c584ece3245be2aa4a5ebb114077d5e3.
[ERR] charts-maimaidx.json | EasyPop - ハッピーシンセサイザ (16) - Single DX Basic (da64667783771d1c465cf96146387b5cdd9f1f8d) | Is duplicate on chartID:da64667783771d1c465cf96146387b5cdd9f1f8d.
[ERR] charts-maimaidx.json | EasyPop - ハッピーシンセサイザ (16) - Single DX Expert (8b69b00ce12042ca768bfa8a0184e833f5d6eba0) | Is duplicate on chartID:8b69b00ce12042ca768bfa8a0184e833f5d6eba0.
[ERR] charts-maimaidx.json | EasyPop - ハッピーシンセサイザ (16) - Single DX Master (615750c5ef4f596912b17de7ce033400a167474d) | Is duplicate on chartID:615750c5ef4f596912b17de7ce033400a167474d.
```

**Comment by zkldi** [comment]
What was the reason behind [forgot happy synthesizer dx, confirmed 勦滅 expert](https://github.com/zkldi/Tachi/pull/1081/commits/a2d857f8f76e9a8145c851d9005298be54a99979)?

Specifically the "forgot happy synthesizer dx" - i don't know how you're updating charts here but i'm alarmed that you could forget to add a chart.

**Comment by Xinnoh** [comment]
I was using Zetaraku as a reference for titles of songs added to buddies plus. On that site, if a DX chart of an old song is released, it groups it with their original chart, hence happy synthesizer DX did not show up among the new songs, instead it was grouped with the original chart's release.

**Comment by zkldi** [comment]
I'm uncomfortable at how by-hand-from-wiki's this whole PR is, but I don't imagine we'll get much better. I'm merging it but I expect you guys to keep an eye on the veracity of the data.

**Pull request merged**